### ISSUE TITLE
Armor returning to inventory

### ISSUE THREAD

**Issue by huntera3** [brandon3055/Draconic-Evolution#258]
I am currently working on a custom modpack (similar to Infinity or Unabridged), which I included DE because I wanted to pit it against the Hardcore Ender Expansion mod. I'm doing a playthrough and noticed that whenever I equip any of the DE armor to my player, the moment I close my inventory, that item is removed from the armor slot and immediately dropped into an open inventory slot. I disabled Inventory tweaks and Travelers Gear to rule those out. Also tested with several other mods armor items. Seems to be isolated to only DE armor items. Tried looking through the configs on Infinity, but didn't see where they had made any customized changes to fix equipment issues with DE. I was able to reproduce this with both the snapshot 5 and 10 versions. Has anyone else seen this happen in their pack? Thanks in advance

**Comment by huntera3** [comment]
I see where another user had reported the same issue and included a video and that issue was closed. Since I'm not certain they were using a custom modpack, and I am, I will attempt to figure out which mod/mods may be causing the issue and report back my findings.

**Comment by brandon3055** [comment]
This is the first i have heard of this issue... And i really cant think of anything on my end that could cause this behavior. I also couldn't find the related issue you mentioned could you please post the issue number. 

The only thing i cam think of is that ether through a bug or deliberately another mod in your back is preventing you from equipping the armor. And i cant imagine it being a bug because that would likely effect other armor. Did you check if anything is printed in the console when this happens?

**Comment by huntera3** [comment]
I think I've found the conflicting mod. With 169 mods installed, it took me a bit to go through all of them adding them back in alphabetical order. And of course it would be Zyin's HUD mod that was causing my particular issue (though I'm not sure if that was the exact cause shadoskill's problem). I guess I'll see if I can drop a message to Zyin to see if he's aware of anything.

As for the previous bug report it was listed as #202 on Dec 17 2015. So wlhlm was spot on with his\her initial assessment of a mod conflict. Just thought it was strange that it was not affecting any of the other armor pieces from other mods during my initial testing and of course it didn't produce any errors in the log that would have made it more obvious- which is what prompted me to start here.

Draconic Evolution is by far one of my favorite mods, so in the meantime, I'll just disable the other mod until I can get with that mod's author. Hopefully though anyone that comes across a similar problem will be able to use this issue report to correct theirs as well.

**Comment by huntera3** [comment]
After getting a bit of sleep, and putting in my eight at work, I decided to do some digging around in the configs or Zyin's mod. Looks like there is an option to automatically unequip armor and tools before they break that can be disabled. So I think somehow Zyins is misinterpreting how Draconic armor and tools indicate NBT data for wear and tear. Hopefully that narrows down the root cause.

And after looking at shadoskill's posted video again, I see what could be the HUD from Zyins on the upper left corner their screen, so I'm nearly certain that is what was causing their items to move into inventory as well. I saw where you dropped Zyin a message on his github. Totally didn't expect that and it shows you're not only engaged with players like myself, but you really do care a great deal about your mod. So thanks, Brandon. Keep up the awesome work. :)
